include/libdnf/rpm/package.hpp

```diff
@@ -38,280 +38,435 @@ class Goal;
 namespace libdnf::rpm {
 
 
-class PackageSetIterator;
-
-
 // IMPORTANT: Package methods MUST NOT be 'noexcept'
 // because accessing deleted sack throws an exception.
 
 
-/// @replaces libdnf/hy-package.h:struct:DnfPackage
-/// @replaces dnf:dnf/package.py:class:Package
+// @replaces libdnf:libdnf/hy-package.h:struct:DnfPackage
+// @replaces dnf:dnf/package.py:class:Package
 class Package {
 public:
-    /// @replaces libdnf/hy-package.h:function:dnf_package_get_identical(DnfPackage * pkg)
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_identical(DnfPackage * pkg)
     bool operator==(const Package & other) const noexcept;
     bool operator!=(const Package & other) const noexcept;
 
-    /// @replaces libdnf/hy-package.h:function:dnf_package_get_id(DnfPackage * pkg)
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_id(DnfPackage * pkg)
     PackageId get_id() const noexcept { return id; };
 
-    /// @replaces dnf:dnf/package.py:attribute:Package.name
-    /// @replaces libdnf/hy-package.h:function:dnf_package_get_name(DnfPackage *pkg);
+    /// @return RPM package Name (RPMTAG_NAME)
```

Comment:
I've just tested that backticks for marking identifiers and code snippets work, I'd use them extensively. e.g. here:
> @return RPM package Name (`RPMTAG_NAME`)

It prints in dark orange in the html docs, very nice :slightly_smiling_face: